macros/udfs/sentiment_analysis.sql

```diff
@@ -0,0 +1,36 @@
+{% macro create_udf_sentiment_analysis() %}
+
+create or replace function {{target.schema}}.udf_sentiment_analysis(text STRING, output INTEGER)
+returns STRING
+language PYTHON
+runtime_version = 3.8
+packages = ('pytorch','transformers','gensim')
+handler = 'sentiment_analysis_py'
+as
+
+$$
+
+from transformers import pipeline
+from gensim.parsing.preprocessing import remove_stopwords
+import string
+
+model = "cardiffnlp/twitter-roberta-base-sentiment-latest"
+-- model = 'nlptown/bert-base-multilingual-uncased-sentiment'
+
+def sentiment_analysis(text, model_id, output='score'):
```

Comment:
your function name (sentiment_analysis) and your handler on L8 have to be equivalent. you also should have the equivalent # of arguments here and in your UDF call.